index.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 const alfy = require('alfy');
+const alfredNotifier = require('alfred-notifier');
```

Comment:
I think that would be great! I can't think of a use case where you want to turn it off as well. It will always be a dependency of my alfy workflows (except for `alfred-updater` which doesn't use `alfy` to start with). Created https://github.com/sindresorhus/alfy/issues/24 to discuss this feature.